internal/storage/request.go

```diff
@@ -3,17 +3,20 @@ package storage
 import (
 	"bytes"
 	"encoding/json"
-	uuid "github.com/satori/go.uuid"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"net/url"
+
+	uuid "github.com/satori/go.uuid"
```

Comment:
Is this something you IDE does or something golint really requires? I mean the re-odering of the imports?

Reply:
I think goimports does the grouping
http://goinbigdata.com/goimports-vs-gofmt/

Reply:
ok, fair enough. I was just wondering. So it is really your IDE, not the linting.